src/parsers/PropertiesParser.js

```diff
@@ -95,31 +95,26 @@ class PropertiesParser extends Parser {
         this.name = "PropertiesParser";
         this.description = "A parser for properties files.";
         this.type = "resource";
-
-        if (!options.filePath) return;
-
-        // Now figure out if we were given a source or a target properties file
-        // If it was a source, just read it. If it was a target, try to figure
-        // out what the source was, and then read that as well.
-
-        this.guessLocaleAndFindSourceFile(options.filePath);
     }
 
     /**
      * @private
      */
-    guessLocaleAndFindSourceFile(filePath) {
+    guessLocaleAndFindSourcePath(sourceFile) {
```

Comment:
It would be nice to annotate param and return types through JSDoc - at least so that TS could give correct hints

Reply:
Done